docs/guides/configure.md

```diff
@@ -146,3 +146,7 @@ Additional kwargs to `compile` functions take higher precedence. So if you set t
   * False disables it for all cases.
   * Integer value enables or disables it depending on the original bit width.
   * With the default value of 8, only the values with original bit width <= 8 will be converted to their original precision.
+* **simulate\_encrypt\_run\_decrypt**: bool = False
+  * Whether to use simulate encrypt/run/decrypt methods of the circuit/module instead of doing the actual encryption/evaluation/decryption.
```

Comment:
I don't think it's that clear. I would say something like:

When this option is set, encrypt and decrypt are void functions, and run is a simple execution. In other words, this option allows to switch off the encryption to quickly test if a function has expected semantic (without paying the price of FHE execution), e.g. during development of the function. It is an unsafe option so is reserved to development time.

Reply:
Sure I'll add it :+1: